### PR TITLE
Add RF 3.0.1, 2.9.2 and 2.8.7 to Travis

### PR DESCRIPTION
**Pull request by aaltat**
We must also test with RF version we support. Updated Travis to
run test for different Robot Framework versions.

### CODE REVIEW

**Comment by aaltat** [comment]
Fixes #751

File: .travis.yml [hunk 1]
```diff
@@ -23,21 +23,31 @@ matrix:
...
     - python: "3.3" # For cron only
       env:
         - BROWSER=firefox
         - SELENIUM=2.53.6
+        - ROBOTFRAMEWORK=3.0.1
 before_script:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
   - pip install selenium==$SELENIUM
+  - pip install selenium==$ROBOTFRAMEWORK
```
Comment (HelioGuilherme66) [inline .travis.yml]: This is wrong. It should be:
`pip install robotframework==$ROBOTFRAMEWORK`

**Review by HelioGuilherme66 — CHANGES_REQUESTED** [review]
We should consider using selenium==2.53.6 with firefox-esr-latest and not with chrome. I suspect that newer chrome does not work well with 2.53.6.

File: .travis.yml (hunk 1 above)
Comment (aaltat) [reply .travis.yml]: Should not play with my hobbies so late in the night and therefore make stupid mistakes like this. Good catch, will fix it.

Comment (aaltat) [reply .travis.yml]: Done

**Pull request merged**